template/hooks/common/functions.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Common functions for cloud hooks.
+
+status=0
+
+env_refresh() {
+
+  echo "Running env_refresh() for environment: $target_env"
+
+  #create drush command
+  drush_cmd="drush @$site.$target_env"
+  echo $drush_cmd
+
+  # Prep for BLT commands.
+  repo_root="/var/www/html/$site.$target_env"
+  export PATH=$repo_root/vendor/bin:$PATH
+  cd $repo_root
+
+  echo "Update starting..."
+  blt deploy:update
```

Comment:
Do you think this is something we need to remediate before this is merged, or can we clean it up later? Like I said, it seems to be working for me as-is, even though I know it's not ideal.